sdk/download.go

```diff
@@ -151,3 +153,40 @@ func DownloadSDK(version string) error {
 	tarURL := TarballURL(version)
 	return DownloadSignedFile(tarFile, tarURL)
 }
+
+// UpdateSignedFile will download will replace the local file if the
+// published signature doesn't match the local copy
+func UpdateSignedFile(file, url string) error {
+	sigFile := file + ".sig"
+	sigURL := url + ".sig"
+
+	// get local sig
+	localSig := []byte{}
+	if _, err := os.Stat(sigFile); err == nil {
+		localSig, err = ioutil.ReadFile(sigFile)
+		if err != nil {
+			return err
+		}
+	}
+
+	// overwrite local sig with latest
```

Comment:
Sounds good. I'll do it to a temp file so as not to copy all the DownloadFile() code. UpdateFile will need some sort of compareFileBytes() function. Either by chunking and comparing bytes or hashing and comparing that.